feat(changelog): elastic-metal-added-🇳🇱-ams-1-opening 2023-02-01 (#1256)
scaleway/docs-content: changelog/february2023/2023-02-01-elastic-metal-added-🇳🇱-ams-1-opening.mdx

---

> title: 🇳🇱 AMS-1 opening

Reviewer: ```suggestion
title: Elastic Metal is now available is AMS-1!
```

> title: Elastic Metal is now available is AMS-1!

Reviewer 2: ```suggestion
title: Elastic Metal is now available in AMS-1!
```

status: added
author:
    fullname: 'Join the #elastic-metal channel on Slack.'
    url: 'https://slack.scaleway.com'
date: 2023-02-01
category: compute
product: elastic-metal
---

> In addition to `PAR-1` and `PAR-2`, Elastic Metal (Lithium range) is now available in **Amsterdam (`AMS-1`)**. More offers will be added progressively in this new Availability Zone.

Reviewer 2: ```suggestion
In addition to `PAR-1` and `PAR-2`, the Elastic Metal Lithium range is now available in **Amsterdam** (`AMS-1`). 
```

> In addition to PAR-1 and PAR-2, Elastic Metal is now available in **Amsterdam (AMS-1)** Availability Zone. The available offers for the launch are the Lithium offers and more will progressively be added in this new AZ.

Reviewer: ```suggestion
In addition to `PAR-1` and `PAR-2`, Elastic Metal (Lithium range) is now available in **Amsterdam (`AMS-1`)**. More offers will be added progressively in this new Availability Zone.
```
